workflow/daymet.py

```diff
@@ -108,10 +115,13 @@ def collectDaymet(bounds, crs, start, end, vars=None, force=False, combine=False
                 dat[var][my_start:my_start+365,:,:] = v
 
     if combine:
+        logging.info(f"Combining all NetCDF into one...")
         import xarray as xr
```

Comment:
Can this be done without xarray?  Currently xarray is not in the list of required packages.  netCDF4 is, however.  If we find we need the xarray or dask libraries for performance, we can add them, but that would require updating documentation, INSTALL, yml files, Docker container (when it exists), etc.